app/src/main/java/com/cherrish/android/presentation/calendar/procedure/ProcedureViewModel.kt

```
@@ -349,3 +509,12 @@ private fun ProcedureUiState.prevStepOrEntry(): PrevResult {
         ProcedureFlow.Entry -> PrevResult.ToEntry
     }
 }
+
+private fun ProcedureUiState.recoveryTargetDateOrNull(): LocalDate? {
+    return try {
+        if (year.isBlank() || month.isBlank() || day.isBlank()) return null
+        LocalDate.of(year.toInt(), month.toInt(), day.toInt())
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)